interface/src/app/node/node.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.node', [
+  'ui.router',
+  'placeholders',
+  'ui.bootstrap'
+])
+
+.config(function config($stateProvider) {
+  $stateProvider.state('node', {
+    url: '/node/{id:int}',
+    views: {
+      main: {
+        templateUrl: 'node/node.tpl.html',
+        controller: 'NodeCtrl as ctrl'
+      }
+    },
+    data: {pageTitle: 'Node Information'}
+  });
+})
+
+.factory('NodeInfo', ['$resource', '$stateParams',
+  function($resource) {
+    return $resource('/api/v0/node/:id');
+  }
+])
+
+.factory('HeavyGenes', ['$resource', function($resource) {
+  return $resource('/api/v0/participation/');
+}])
+
+.controller('NodeCtrl', ['NodeInfo', '$stateParams', '$log',
+  function NodeController(NodeInfo, $stateParams, $log) {
+    var self = this;
+    if (!$stateParams.id) {
+      self.statusMessage = 'Please specify node ID in the URL.';
+      return;
+    }
+    self.id = $stateParams.id;
```

Comment:
**self** is used here to denote that all of its properties will be accessible from the HTML that is under control of this controller. This usage is more like a coding style that is bundled with **controllerAs** syntax. Does this make sense?